src/pages/createOrder.js

```diff
@@ -472,6 +485,30 @@ class CreateOrderPage extends React.Component {
     };
   }
 
+  getDefaultCustomFields() {
+    const { customFields } = this.state;
+    let { customFieldsDefaultValues } = this.props;
+
+    try {
+      // `customFieldsDefaultValues` is from the url data querystring
+      // it is needed as an object hence the need to parse
+      customFieldsDefaultValues = JSON.parse(customFieldsDefaultValues);
```

Comment:
Sorry, I missed the part where you decided to structure the data as JSON in the URL. Let's not do that for now.

I initially proposed query params with a prefix such as `data[jsonUrl]`

Reply:
I thought this was standard, but if it's not, your approach is valid.

Reply:
Check out my latest commit.